stat/python_script_lm_predictors.py

```diff
@@ -0,0 +1,56 @@
+import numpy
+from keras.models import Sequential
+from keras.layers import Dense
+import keras.optimizers
+import matplotlib.pyplot as plt
+import random
+
+
+# load the dataset
+train=numpy.loadtxt('train.csv', delimiter=',',skiprows=1)
+test=numpy.loadtxt('test.csv', delimiter=',',skiprows=1)
+
+# split into input and output variables
+train_input=train[:,numpy.array([5,6,13])]
+train_output=train[:,1]
+
+test_input=test[:,numpy.array([5,6,13])]
+test_output=test[:,1]
+
+random.seed(2) # Pro opakovatelnost výsledků
+
+# define the keras model
+model=Sequential()
+model.add(Dense(3, activation='relu'))
+model.add(Dense(2, activation='relu'))
+model.add(Dense(1, activation='relu'))
+
+
+
+# compile the keras model
+
+opt = keras.optimizers.Adam(learning_rate=0.001)
+
+model.compile(loss='mae', optimizer=opt, metrics=['mae'])
+
+# fit the keras model on the dataset
+
+m=model.fit(train_input,train_output,epochs=30000,verbose=1,steps_per_epoch=None)
```

Comment:
Z mých zkušeností je takto "velký" parametr learning rate u kompilátoru Adam v kombinaci s daným počtem epoch teměř jistota přeučení. 
Pokud to funguje - ok. Jen bych radeji videl nejake grafy prubehu uceni (TensorBoard)